modules/library/api/src/main/java/org/geotools/styling/PolygonSymbolizer.java

```diff
@@ -2,7 +2,7 @@
  *    GeoTools - The Open Source Java GIS Toolkit
  *    http://geotools.org
  * 
- *    (C) 2002-2008, Open Source Geospatial Foundation (OSGeo)
+ *    (C) 2002-2016, Open Source Geospatial Foundation (OSGeo)
```

Comment:
Mind, this means you don't need to change the commit either, leave them be, just remember that you don't need to alter them anymore for the future changes.